content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -69,8 +69,8 @@ integrates with them, you must pass additional arguments to the
 {{< text bash >}}
     $ helm template \
         --set kiali.enabled=true \
-        --set "kiali.dashboard.jaegerURL=http://$(kubectl get svc tracing -o jsonpath='{.spec.clusterIP}'):80" \
-        --set "kiali.dashboard.grafanaURL=http://$(kubectl get svc grafana -o jsonpath='{.spec.clusterIP}'):3000" \
+        --set "kiali.dashboard.jaegerURL=http://$(kubectl get svc tracing -n istio-system -o jsonpath='{.spec.clusterIP}'):80" \
+        --set "kiali.dashboard.grafanaURL=http://$(kubectl get svc grafana -n istio-system -o jsonpath='{.spec.clusterIP}'):3000" \
         install/kubernetes/helm/istio \
         --name istio --namespace istio-system > $HOME/istio.yaml
```

Comment:
Also change `--namespace istio-system` to keep consistent?

Reply:
Check the latest PR :-)